lua/astrocommunity/editing-support/yanky-nvim/init.lua

```diff
@@ -0,0 +1,40 @@
+return {
+  "gbprod/yanky.nvim",
+  dependencies = { { "kkharji/sqlite.lua", enabled = not jit.os:find "Windows" } },
+  opts = function()
+    local mapping = require "yanky.telescope.mapping"
+    local mappings = mapping.get_defaults()
+    mappings.i["<c-p>"] = nil
+    return {
+      highlight = { timer = 200 },
+      ring = { storage = jit.os:find "Windows" and "shada" or "sqlite" },
+      picker = {
+        telescope = {
+          use_default_mappings = false,
+          mappings = mappings,
+        },
+      },
+    }
+  end,
+  keys = {
+        -- stylua: ignore
```

Comment:
Why do you have this here?

Reply:
Otherwise it will be formatted to multiple lines per keymap. By disabling lua formatting and keep each keymap in one line, the readability is way better imo.

Reply:
We have other cases where we let stylua split the line, so I'd say we should keep doing it. Can you remove the ignore line here, and we're GTG 🚀

Reply:
Alright 👌✅

Reply:
Just formats the first line, so nevermind actually 😅